plugin/pkg/kubectl/client/plugin_client.go

```diff
@@ -0,0 +1,192 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
Not 100% sure this is the file for this, but... it might be good to add a comment about how some of this will be removed when your CLI PR is merged and we have a common library that we can leverage - and a URL to the PR too. Just so people can make the linkage and see where we're headed.